addons/l10n_jo_edi/tests/test_edi_jo.py

```diff
@@ -0,0 +1,336 @@
+from odoo import Command
+from odoo.tests import tagged
+from odoo.tools import misc
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestEdiJo(AccountTestInvoicingCommon):
+    @classmethod
+    def setUpClass(cls, chart_template_ref='jo_standard'):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+        cls.company = cls.setup_company_data('Jordan Company', chart_template=chart_template_ref)['company']
+        cls.company.vat = '8000514'
+
+        def _create_tax(amount, amount_type):
```

Comment:
plz use existing taxes in the coa. Your tests must reflect the reality

Reply:
The thing is, I will need to create taxes at some point anyway. That is because the special taxes (which are of fixed amount) are not introduced in the CoA. Is it fine if I create some taxes and use the rest from the CoA ?